frontend/ui/data/keyboardlayouts/ja_keyboard_keys.lua

```diff
@@ -47,19 +47,19 @@ return {
     l_1 = { label = "@-_/", alt_label = "１",
            "@", west = "-", north = "_", east = "/", south = "１" },
     l_2 = { label = "abc", alt_label = "２",
-           "a", west = "b", north = "c", east = "２" },
+           "a", west = "b", north = "c", south = "２" },
```

Comment:
Are these "full width CJK" digits ?
If yes, it's just there as some additional digits ? Or are these the main digits a user may input ? If yes, I guess there are many issues, like entering a page number in the GoToPage widget, that would not work.

Reply:
These are full-width digits, but there's a separate "digit" keypad which uses half-width (regular) digits. If someone really wants to input digits they'd probably use the digit keypad.